Invoking command in '/home/sazma/ws_future/src/mpu_6050/mpu_py': PYTHONPATH=/home/sazma/ws_future/build/mpu_py/prefix_override:/home/sazma/ws_future/install/mpu_py/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../../build/mpu_py build --build-base /home/sazma/ws_future/build/mpu_py/build install --record /home/sazma/ws_future/build/mpu_py/install.log --single-version-externally-managed
Invoked command in '/home/sazma/ws_future/src/mpu_6050/mpu_py' returned '0': PYTHONPATH=/home/sazma/ws_future/build/mpu_py/prefix_override:/home/sazma/ws_future/install/mpu_py/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../../build/mpu_py build --build-base /home/sazma/ws_future/build/mpu_py/build install --record /home/sazma/ws_future/build/mpu_py/install.log --single-version-externally-managed
